src/test/java/uk/gov/hmcts/reform/hmc/service/HearingManagementServiceTest.java

```diff
@@ -956,6 +969,44 @@ private void mockDeleteRequest() {
     @Nested
     @DisplayName("updateHearing")
     class UpdateHearing {
+        public void setUp() {
```

Comment:
we should have an @BeforeEach on the setup method

Reply:
Done